pages/live.tsx

```diff
@@ -0,0 +1,203 @@
+import { InferGetStaticPropsType } from "next";
+import { useState } from "react";
+import { BsArrowRightCircle } from "react-icons/bs";
+import { useTina } from "tinacms/dist/react";
+import { client } from "../.tina/__generated__/client";
+import { BuiltOnAzure } from "../components/blocks/builtOnAzure";
+import { VideoEmbed } from "../components/blocks/videoEmbed";
+import { UtilityButton } from "../components/button/utilityButton";
+import { Layout } from "../components/layout";
+import { LiveHeader } from "../components/live/header";
+import { PresenterLink } from "../components/usergroup/presenterLink";
+import { Container } from "../components/util/container";
+import { SEO } from "../components/util/seo";
+import { VideoCard } from "../components/util/videoCards";
+import { getEvents, getSpeakersInfoFromEvent } from "../services/server/events";
+import { getYoutubePlaylist } from "../services/server/youtube";
+
+export default function LivePage(
+  props: InferGetStaticPropsType<typeof getStaticProps>
+) {
+  const { data } = useTina({
+    data: props.data,
+    query: props.query,
+    variables: props.variables,
+  });
+  const [visibleVideos, setVisibleVideos] = useState(9);
+
+  return (
+    <Layout>
+      <SEO seo={data.live.seo} />
+      <LiveHeader {...data.live} />
```

Comment:
Use explicit props once you've created the type for the props

Reply:
Done